packages/template/src/providers/stack-provider-client.tsx

```diff
@@ -29,7 +31,7 @@ export function StackProviderClient(props: {
 export function UserSetter(props: { userJsonPromise: Promise<CurrentUserCrud['Client']['Read'] | null> }) {
   const app = useStackApp();
   useEffect(() => {
-    const promise = (async () => await props.userJsonPromise)();  // there is a Next.js bug where Promises passed by server components return `undefined` as their `then` value, so wrap it in a normal promise
+    const promise = (async () => await props.userJsonPromise)(); // there is a Next.js bug where Promises passed by server components return `undefined` as their `then` value, so wrap it in a normal promise
```

Comment:
why? I usually put two spaces between code and comments for readability (maybe we can have ESLint enforce this)
```suggestion
    const promise = (async () => await props.userJsonPromise)(); // there is a Next.js bug where Promises passed by server components return `undefined` as their `then` value, so wrap it in a normal promise
```